app/src/main/java/com/mealsmadeeasy/data/FakeMealStore.kt

```diff
@@ -164,4 +166,41 @@ class FakeMealStore : MealStore {
             else -> return Single.just(iceCream)
         }
     }
+
+    override fun getRecipe(id: String): Single<Recipe> {
+        val tacosRecipe = Recipe(
+                prepTime = 20,
+                steps = listOf(
+                        RecipeStep("Cut chicken into 1/2 inch strips."),
+                        RecipeStep("Heat oil in large skillet over medium-high heat, adding chicken once hot. " +
+                                "Cook and stir for 10 minutes or until cooked through."),
+                        RecipeStep("Spoon 1/3 of the chicken into each tortilla and sprinkle with cheese. " +
+                                "Top evenly with lettuce and tomato.")
+                ),
+                ingredients = listOf(
+                        Ingredient(id = "taco-shell", name = "Flour tortilla shells", quantity = 3, unitName = "Shells", isMeasurable = false),
+                        Ingredient(id = "taco-meat_chicken", name = "Chicken", quantity = 0.5, unitName = "lbs"),
+                        Ingredient(id = "shredded_cheese", name = "Shredded cheese", quantity = 0.75, unitName = "Cups"),
+                        Ingredient(id = "lettuce", name = "Lettuce", quantity = 2, unitName = "Leaves"),
```

Comment:
This reads kinda awkwardly as "2 Leaves Lettuce." It's going to be effectively impossible to contextually add the word "of" in between when it's appropriate (and this is awful for localization). So I'd either be cheaty and change this to shredded lettuce, or make the ingredient name "Lettuce leaves" instead, keep the unit name as-is, and mark it as immesurable.